README.md

```diff
@@ -260,6 +260,39 @@ npm run code
 npm test
 ```
 
+## Plugins
+
+Sinon chrome module supports by plugins.
+
+### Cookie plugin
+
+You can test chrome cookie behaviour.
+
+#### install
+
+```js
+var chrome = require('sinon-chrome');
+var cookiePlugin = require('sinon-chrome/out/plugins/cookies');
```

Comment:
Extending with plugins is cool idea. But in this place it looks not very elegant because of `out` in path